docs/source/install.rst

```diff
@@ -1,15 +1,39 @@
 Installing
 ==========
 
-Here are two ways to hack on python.org:
+Here are three ways to hack on python.org:
 
-1. :ref:`vagrant-setup`
-2. :ref:`manual-setup`
+1. :ref:`docker-setup`
+2. :ref:`vagrant-setup`
+3. :ref:`manual-setup`
```

Comment:
I would like a alphabetic enumeration :-). First docker, second manual setup and finally vagrant setup

Just an opinion

Reply:
Do you mean the following?
```
A. Docker Setup
B. Manual Setup
C. Vagrant Setup
```

If so, as long as @berkerpeksag is okay with it I don't mind.

Reply:
Yes, that I mean :-)